OpenRA.Mods.Common/Widgets/ObserverProductionIconsWidget.cs

```diff
@@ -56,7 +56,7 @@ public ObserverProductionIconsWidget(World world, WorldRenderer worldRenderer)
 			this.world = world;
 			this.worldRenderer = worldRenderer;
 			clocks = new Dictionary<ProductionQueue, Animation>();
-			timestep = world.IsReplay ? world.WorldActor.Trait<MapOptions>().GameSpeed.Timestep : world.Timestep;
+			timestep = world.Timestep;
```

Comment:
Can we please remove `timestep` completely and use `world.Timestep` directly at the one place in `Draw()`?